server/services/v1/workers/worker.go

```diff
@@ -101,10 +102,22 @@ func (w *Worker) jitterSleep() {
 }
 
 func (w *Worker) Start() {
+	defer catchWorkerPanics()
 	w.jitterSleep()
 	w.Loop()
 }
 
+func catchWorkerPanics() {
```

Comment:
This is good to have, but is it still happening? Or is this just in case to catch any bugs in the future?

Reply:
This is defensive to catch any bugs in the future.